antismash/common/gff_parser.py

```diff
@@ -21,6 +22,40 @@
 MODIFY_LOCATIONS_BY_PHASE = False
 
 
+_BIOPYTHON_STRAND = {'+': 1,
+                     '-': -1,
+                     '.': 0}
+
+
+def to_seqfeature(feature: gffutils.feature.Feature) -> SeqFeature:
+    """
+        Converts a gffutils.Feature object to a Bio.SeqFeature object.
+
+        The GFF fields `source`, `score`, `seqid`, and `frame` are stored as
+        qualifiers.  GFF `attributes` are also stored as qualifiers.
+
+        Parameters
+        ----------
+        feature : Feature object, or string
+            If string, assume it is a GFF or GTF-format line; otherwise just use
+            the provided feature directly.
+    """
+    qualifiers = {
+        "source": [feature.source],
+        "score": [feature.score],
+        "seqid": [feature.seqid],
+        "frame": [feature.frame],
+    }
+    qualifiers.update(feature.attributes)
+    start, stop = sorted((feature.start, feature.end))
+    return SeqFeature(
+        SimpleLocation(start - 1, stop, strand=_BIOPYTHON_STRAND[feature.strand]),
+        id=feature.id,
+        type=feature.featuretype,
+        qualifiers=qualifiers,
+    )
+
+
```

Comment:
This is almost an exact copy/paste of `gffutils.biopython_integration.to_seqfeature`. You've removed the string handling portion and have added type annotations, but have left the docstring saying that a string is accepted. I don't know why you have done this instead of just using the existing function from the library you've just added as a dependency.

This is a licensing minefield. Before doing anything else with this PR, we will need to know what else you've copied from other repositories.

Reply:
Sure, there is almost the same between `gffutils.biopython_integration.to_seqfeature` and this function, but in `gffutils`, it checked `FeatureLocation`, which was once deleted in biopython and marked as `OBSOLETE` now.

https://github.com/daler/gffutils/blob/67a4cd7edc11bb2d1ff97d8c777fb1e6006296e5/gffutils/biopython_integration.py#L6C1-L11C68

Reply:
That doesn't change potential licensing issues. 

> This is a licensing minefield. Before doing anything else with this PR, we will need to know what else you've copied from other repositories.

Please, address the above.

Reply:
Sure. I'm sorry for this licensing minefield.

I'm sure that none of the other codes were copied or modified from elsewhere. For test data, I modified the example of bacteriophage f1 J02448.